setup.py

```diff
@@ -105,7 +102,7 @@ def get_extensions():
         return []
 
     def _cython_modules():
-        package_root = os.path.dirname(__file__)
+        os.path.dirname(__file__)
```

Comment:
How about deleting this line?

Reply:
I've focused on the only linting error. but it should be removed. thanks!